cli/README.md

```diff
@@ -72,6 +72,21 @@ To add user :
 run `node cli.mjs --addUser <userId>`
 example: node cli.mjs --addUser "123~A#netflix"
 
+## Downloading external git repository
+
+To download/clone external github repository and save repository contents in MF, below options can be used :
+
+`--downloadOverrides` - specify the url of a github repository to clone
+
+`--overrideLocation` - Specify the location relative to current working directory, in which to save the repository's contents. By default, location is set to `mock-firebolt/cli/externalOverrides`
+
+run `node cli.mjs --downloadOverrides <URL of git repository> --overrideLocation <location to save contents>`
+
+examples: 
+node cli.mjs --downloadOverrides https://github.comcast.com/lightning-automation/firebolt-cert-test-data.git
```

Comment:
We can't use Comcast information. This example should be generic. It doesn't have to be a real repo:

"https://github.com/myOrg/myRepo.git"